cluster-autoscaler/push_image.sh

```diff
@@ -20,15 +20,21 @@ if [ -z $IMAGE_TO_PUSH ]; then
   exit 1
 fi
 
+if [[ "${IMAGE_TO_PUSH}" == "gcr.io/"* ]]; then
```

Comment:
Handle `k8s.gcr.io` as well.

Reply:
See #512

Reply:
@mwielgus Thanks. Fixed in latest commit. PTAL